test/cmd/authentication.sh

```diff
@@ -47,7 +47,7 @@ os::cmd::expect_success "oc policy can-i --list"
 whoamitoken="$(oc process -f "${OS_ROOT}/test/testdata/authentication/scoped-token-template.yaml" TOKEN_PREFIX=whoami SCOPE=user:info USER_NAME="${username}" USER_UID="${useruid}" | oc create -f - -o name | awk -F/ '{print $2}')"
 os::cmd::expect_success_and_text "oc get user/~ --token='${whoamitoken}'" "${username}"
 os::cmd::expect_success_and_text "oc whoami --token='${whoamitoken}'" "${username}"
-os::cmd::expect_failure_and_text "oc get pods --token='${whoamitoken}' -n '${project}'" "prevent this action; User \"scoped-user\" cannot list pods in project \"${project}\""
+os::cmd::expect_failure_and_text "oc get pods --token='${whoamitoken}' -n '${project}'" "pods is forbidden: User \"scoped-user\" cannot list pods in the namespace \"${project}\""
```

Comment:
I'm fine with it, but @liggitt this takes from project to namespace in the text.  I'm betting it also changes the text for a forbidden project request.

Reply:
We already include the kube forbidden message since we use their request handlers.  Saying just namespace is less confusing than saying both namespace and project as we do today.

Reply:
@deads2k beats having duplicated text, we have "no way" to not make errors still print namespace anyway

Reply:
> @deads2k beats having duplicated text, we have "no way" to not make errors still print namespace anyway

I'm not arguing against it. I'm pointing out the difference so that no one claims they're surprised.  I wasn't the guy who wanted it to begin with.  That person liked users more.

Reply:
> this takes from project to namespace in the text

/shrugs

```
oc options 
...
-n, --namespace='': If present, the namespace scope for this CLI request
```